pyveda/frameworks/transforms.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+import scipy.ndimage as ndi
+
+def rescale_toa(arr, dtype=np.float32):
+    """
+    Rescale any multi-dimensional array of shape (D, M, N) by first subtracting
+    the min from each (M, N) array along axis 0, then divide each by the
+    resulting maximum to get (float32) distributions between [0, 1].
+    Optionally spec uint8 for 0-255 ints.
+    """
+    # First look at raw value dists along bands
+
+    arr_trans = np.subtract(arr, arr.min(axis=(1, 2))[:, np.newaxis, np.newaxis])
+    arr_rs = np.divide(arr_trans, arr_trans.max(axis=(1, 2))[:, np.newaxis, np.newaxis])
+    if dtype == np.uint8:
+        arr_rs = np.array(arr_rs*255, dtype=np.uint8)
+    return arr_rs.T
+
+
+def bands_subset_f(arr, band_numbers):
+    """
+    Subset array to array of  just given band indices
+    arr: float32
+    band_numbers: List. List of band numbers to subset
+    """
+    return arr[band_numbers, ...]
+
+# Augmentation Functions Definition- written by Bernardo Henz
+# https://github.com/bernardohenz/ExtendableImageDatagen/blob/master/util/extendable_datagen.py
+
+
+def apply_transform(x, transform_matrix, channel_index=0, fill_mode='nearest', cval=0.):
```

Comment:
it could be nice to have some docs on these functions. Can we figure out what they do so people will know if they want to use them?

Reply:
docs added